addon/components/action-button.js

```diff
@@ -23,7 +23,6 @@ export default FrostButton.extend({
     if (this.get('type') === 'confirm') {
       if (this.attrs['onConfirm']) {
         this.attrs['onConfirm']()
-        this.attrs['onClose']()
```

Comment:
This should remain the default.  What I would like to do instead is add a boolean closeOnClick (default to true) that can be set to false in the case when you don't want the default behavior.